Changelog.md

```diff
@@ -3,6 +3,7 @@
 ## master (unreleased)
 
 - Added isolated tests for shifting mechanics in USA calendars - previously untested (#335).
+- Added several one-off public holidays to UK calendar.
```

Comment:
could you please add a reference to the github issue here?

Reply:
Done. I've also rebased my branch.